m3u8/reader_test.go

```diff
@@ -862,6 +863,73 @@ func TestMediaPlaylistWithSCTE35Tag(t *testing.T) {
 	}
 }
 
+func TestDecodeLowLatencyMediaPlaylist(t *testing.T) {
+	is := is.New(t)
+	f, err := os.Open("sample-playlists/media-playlist-low-latency.m3u8")
+	is.NoErr(err) // must open file
+	p, listType, err := DecodeFrom(bufio.NewReader(f), true)
+	is.NoErr(err) // must decode playlist
+	pp := p.(*MediaPlaylist)
+	CheckType(t, pp)
+	is.Equal(listType, MEDIA) // must be media playlist
+	// check parsed values
+	is.Equal(pp.TargetDuration, uint(4))
```

Comment:
Always add a comment to the right of the `is.Equal()` calls. 

The output when running a nice tool like `gotestsum` is much nicer with such a comment. Compare (lacking the colors) how the message is much better with a comment.

gotestsum
✖  m3u8 (294ms)

=== Failed
=== FAIL: m3u8 TestDecodeLowLatencyMediaPlaylist (0.00s)
    structure_test.go:13: *m3u8.MediaPlaylist implements Playlist interface OK
        reader_test.go:876: 4 != 5

with 

gotestsum
✖  m3u8 (323ms)

=== Failed
=== FAIL: m3u8 TestDecodeLowLatencyMediaPlaylist (0.00s)
    structure_test.go:13: *m3u8.MediaPlaylist implements Playlist interface OK
        reader_test.go:876: 4 != 5 // target duration must be 5